grr/server/grr_response_server/gui/validate_iap.py

```diff
@@ -0,0 +1,98 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
One question: GRR is already coverted by the Apache license (https://github.com/google/grr/blob/master/LICENSE). Is there any specific reason to insert the license header into this file?

Reply:
This may be a misinterpretation on my part, but this mini-library of functions is taken from the Google sample code on validating IAP headers. I placed this here to emphasize its distinct status, but I may be violating some standards here. Do you think we should replace this blob?

Reply:
Given that GRR is also a Google project completely covered by the same license, I think it's safe to assume that an explicit copy of the license is not needed here.